### PR TITLE
Changelog Update and getting back changes undone to resolve merge conflicts

### LINKED ISSUE #24: complete current the 2-part pull-request & get tests working
* [ ]  increment version to 1.0.0
* [ ] add a note to the changelog https://github.com/iobis/pyobis/blob/master/Changelog.rst

### PR DESCRIPTION
**Pull request by ayushanand18**
# Overview of Changes
This pull request tries to resolve #24 

With this PR we have got three objectives accomplished:
+ Getting back changes which, we had to undo to resolve merge conflicts in #23 
+ We have also dropped obisDownload as discussed in the call.
+ Updating the Changelog to v1.0.0, and added description of changes

# Notes for reviewers
The tests are passing for this PR with nearly 84% overall coverage and 100% coverage for all modules.

# Next Steps
I am now working on Contributing guidelines.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -118,8 +113,8 @@ Meta
 .. |docs| image:: https://readthedocs.org/projects/pyobis/badge/?version=latest
    :target: http://pyobis.readthedocs.org/en/latest/?badge=latest
 
-.. |tests| image:: https://github.com/iobis/pyobis/actions/workflows/tests.yml/badge.svg
+.. |tests| image:: https://github.com/ayushanand18/pyobis/actions/workflows/tests.yml/badge.svg
    :target: https://github.com/iobis/pyobis/actions/workflows/tests.yml
 
 .. |coverage| image:: https://coveralls.io/repos/sckott/pyobis/badge.svg?branch=master&service=github
-   :target: https://coveralls.io/github/sckott/pyobis?branch=master
```
Comment (7yl4r) [inline README.rst:125]: This is pointing to the wrong repo. Need to update the url or remove the coveralls badge.

**Review by 7yl4r — CHANGES_REQUESTED** [review]
Looks good. Two minor changes:

* [ ] need to update __version__ in __init__
* [ ] need to update the coveralls badge URL

As a side note: I want to get tests running on PRs too. I think that requires only a small change to tests.yml.

File: README.rst [hunk 2]
```diff
@@ -118,8 +113,8 @@ Meta
 .. |docs| image:: https://readthedocs.org/projects/pyobis/badge/?version=latest
    :target: http://pyobis.readthedocs.org/en/latest/?badge=latest
 
-.. |tests| image:: https://github.com/iobis/pyobis/actions/workflows/tests.yml/badge.svg
+.. |tests| image:: https://github.com/ayushanand18/pyobis/actions/workflows/tests.yml/badge.svg
```
Comment (ocefpaf) [inline README.rst:116]: Please point all status check to the upstream repo and not your fork.

File: README.rst (hunk 1 above)
Comment (ayushanand18) [reply README.rst:125]: Thanks @7yl4r for the review! I will update the URL.

File: README.rst (hunk 2 above)
Comment (ayushanand18) [reply README.rst:116]: Thank you @ocefpaf for the review! I am sorry I just missed updating this, I was testing whether the badge works or not but forgot to update it later. I'll make the necessary changes.

File: README.rst (hunk 1 above)
Comment (ayushanand18) [reply README.rst:125]: I couldn't find a way to update this a coveralls.io badge, maybe we need to register this repo at coveralls.

**Pull request merged**